cmd/serve.go

```diff
@@ -496,6 +503,14 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 			}
 			serveOpts.AnchorPlatformAPIService = apAPIService
 
+			// Setup Distribution Account Resolver
+			distAccResolverOpts.AdminDBConnectionPool = adminDBConnectionPool
+			distAccResolver, err := signing.NewDistributionAccountResolver(distAccResolverOpts)
```

Comment:
❓Any reason why we're not using DI here ? 
```golang
distAccResolver, err := di.NewDistributionAccountResolver(ctx, distAccResolverOpts)
```

Reply:
Thanks for catching that. FIxed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/212/commits/06a131833b1a60dedf6a9dde092cfe1f3bf79a86.